src/r4ge.py

```diff
@@ -191,14 +191,14 @@ def printer():
     # static mode: print stdin and open ipython shell to perform manual concretization 
 
     # print stdin
-    if state_found is not None:
+    if state_found != None:
```

Comment:
be careful, refer to object identity and object equality in python
comparing against None is preferred to be done by "is" operator